spec/components/parameters/query_filter_generic_created_after.yaml

```diff
@@ -0,0 +1,7 @@
+name: created_after
+in: query
+schema:
+  description: It allows to filter items by create date after a given date.
+  type: string
+  format: date-time-iso8601
+  example: 2018-01-15T00:00:00+00:00
```

Comment:
@pack7df please add an empty line at the end of each file, check if you can change the configuration on your editor